src/electron/views/screenshot/screenshot-view.tsx

```diff
@@ -1,13 +1,31 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import { NamedFC } from 'common/react/named-fc';
+import { ScreenshotData } from 'common/types/store-data/unified-data-interface';
+import { isEmpty, isUndefined } from 'lodash';
 import * as React from 'react';
 
-export type ScreenshotViewProps = {};
-export const ScreenshotView = NamedFC<ScreenshotViewProps>('ScreenshotView', props => {
-    return (
-        <div>
-            <h1 tabIndex={0}>This is the screenshot view</h1>
-        </div>
-    );
+import { Screenshot } from './screenshot';
+
+export type ScreenshotViewProps = {
+    screenshotData: ScreenshotData;
+};
+
+const screenshotAltText: string = 'axe-android results screenshot with highlighted components';
+
+export const ScreenshotView = NamedFC<ScreenshotViewProps>('ScreenshotView', (props: ScreenshotViewProps) => {
+    if (isUndefined(props.screenshotData) || isEmpty(props.screenshotData)) {
+        return (
+            <div>
```

Comment:
is it absolutely necessary to have this sctructure? `<div>` > `<p>`?

Reply:
It's not; removed the <p> tags